a/antlr4/Dockerfiles/ubi-8.5/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM registry.access.redhat.com/ubi8 
+
+WORKDIR /opt/antlr4
+
+ARG ANTLR_VERSION="4.11.1"
```

Comment:
By defualt we send the `$PACKAGE_VERSION` argument to our build script and Docker file.
So in this case, we never send ANTLR_VERSION, hence it always build 4.11.1 version.

To fix this, either assign ANTLR_VERSION with `$PACKAGE_VERSION` in `build_info.json` file in `args` section or rename ANTLR_VERSION arg to PACKAGE_VERSION.